src/Accordion/Accordion.jsx

```diff
@@ -0,0 +1,31 @@
+import { useState, useEffect, Fragment } from "react";
+import AccordionItem from "../AccordionItem/AccordionItem";
+import "./Accordion.scss";
+
+const Accordion = () => {
+  const [data, setData] = useState([]);
+
+  useEffect(() => {
+    requestAccordion();
+  });
+
+  async function requestAccordion() {
+    const result = await fetch(`https://jsonplaceholder.typicode.com/posts`);
+    const resultJson = await result.json();
+    setData(resultJson);
+  }
+  return (
+    <>
+      <h1 className="main-title">How can we help you?</h1>
+      {data.map(({ title, body, id }) => {
+        return (
+          <Fragment key={id}>
```

Comment:
fragment is extra here. you can put key on AccordionItem. Please send a voice to explain when we use fragment.

Reply:
Done.